src/core/issues-notes.js

```diff
@@ -23,7 +23,18 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
+    Array.from($ins)
+      .filter(({ dataset: { number } }) => number && !ghIssues[number] && conf.githubAPI)
+      .forEach(elem => {
+        const { dataset: { number } } = elem;
+        const msg = `Github issue ${number} was closed on GitHub, so removing from spec`;
+        pub("warn", msg);
+        elem.remove();
+      });
     $ins.each(function(i, inno) {
+      if (!inno.parentNode) {
```

Comment:
Nit: as this `if` is "filtering", can you do something like this here (I don't know how to do it in jQeury, but...):
```
$ins
  .filter((i, elem) => elem.parentNode)
  .each(function(i, inno) { 
```

We should also rename "inno"... I don't know who named it that, but it's not a great name.

Reply:
`$ins` is actually in normal JavaScript that is why we do `$inno = $(inno)`. So elem.parentNode works, tested this just now things do work.

Reply:
Made changes except that I forgot to change the name of `inno`

Reply:
I have renamed inno to elem. Is this better ?